### DISCUSSION TITLE
How to avoid processing a .jinja file?

### DISCUSSION THREAD

**Discussion by AntoineD** [copier-org/copier#1555] [General]
Without changing any extension, is there a way to prevent copier from processing a `.jinja` file but still copy it?

**Comment by pawamoy** [comment]
Wrap the whole file contents with `{% raw %}...{% endraw %}`. The `.jinja` extension will be removed though.

**Reply by AntoineD** [reply]
I guess the only choice is to change the extension then, right?

**Reply by pawamoy** [reply]
I suppose you could double the extension :sweat_smile: `filename.jinja.jinja`

**Reply by AntoineD** [reply]
Nice, I'll try that!

**Reply by AntoineD** [reply]
Unfortunately that removes all the non-copier related jinja templating!

**Reply by pawamoy** [reply]
I'm not sure to understand, can you illustrate what you mean with a small example? Have you wrapped your file's contents in `{% raw %}...{% endraw %}`?

**Reply by AntoineD** [reply]
I forgot the `raw` guards, it works fine thank you.

**Comment by yajo** [comment]
Use a different extension such as ˋ.j2ˋ for the templates that copier has to actually process, and configure it with https://copier.readthedocs.io/en/stable/configuring/#templates_suffix
